content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,256 @@
+      usernamePrefix: "-"                        # The prefix that is prepended to username claims to prevent clashes with existing names
+      groupsClaim: "groups"                      # The JWT claim that the provider uses to return your groups
+      groupsPrefix: ""                          # The prefix that is prepended to group claims to prevent clashes with existing names
+      requiredClaims:                            # The key value pairs that describe required claims in the identity token
+  ```
+
+<br />
+
+
+2. Under the ``clientConfig`` parameter section of Kubernetes pack, uncomment the ``oidc-`` configuration lines. <br /><br />
+
+  ```
+  clientConfig:
+    oidc-issuer-url: "{{ .spectro.pack.kubernetes-eks.managedControlPlane.oidcIdentityProvider.issuerUrl }}"
+    oidc-client-id: "{{ .spectro.pack.kubernetes-eks.managedControlPlane.oidcIdentityProvider.clientId }}"
+    oidc-client-secret: 1gsranjjmdgahm10j8r6m47ejokm9kafvcbhi3d48jlc3rfpprhv
+    oidc-extra-scope: profile,email
+  ``` 
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Azure AKS" key="Azure AKS">
+ 
+
+Kubernetes RBAC is enabled by default during AKS cluster creation. Follow these steps to link users and Roles to Azure AD. 
+
+<br />
+
+1. In the Azure console, verify Kubernetes RBAC is enabled for your cluster. If RBAC wasn't enabled when you originally deployed your cluster, you'll need to delete and re-create your cluster.
+<br />
+
+2. In the Azure console, create groups and users in Azure AD, then create Roles and RoleBindings for the users.
+<br />
+
+3. In Palette, create a cluster profile using the Spectro RBAC add-on pack available under Authentication packs in the Public Repo. Check out [Creating Cluster Profiles](/cluster-profiles/task-define-profile) to learn about cluster profiles.
+<br />
+
+4. Add information in the Spectro RBAC pack for users and groups and their Roles and RoleBindings based on the same information in Azure AD: <br /><br /> 
+
+  ```
+  clusterRoleBindings:
+      - role: admin
+        name: special-override-name
+        subjects:
+        - {type: User, name: user5}
+        - {type: Group, name: group5}
+      - role: view
+        subjects:
+        - {type: User, name: user6}
+        - {type: Group, name: group6}
+        - {type: ServiceAccount, name: group6, namespace: foo}
+      namespaces:
+      - namespace: team1
+        createNamespace: true
+        roleBindings:
+      - role: admin
+        name: special-override-name-role
+        kind: ClusterRole
+        subjects:
+        - {type: User, name: user3}
+        - {type: Group, name: group3}
+      - role: view
+        kind: ClusterRole
+        subjects:
+        - {type: User, name: user4}
+        - {type: Group, name: group4}
+  ```
+<br /> 
+
+
+5. To get the user-specific kubeconfig file, run the following command:
+
+  ``az aks get-credentials --resource-group <resource-group> --name <cluster-name>``
```

Comment:
@MGamya @arvind-sys
The above command is in the original documentation. 
- Why would users run the command? 
- What does the command do?

Reply:
Question no longer applies - Azure AKS tab is removed.